api/sse/sse_service.py

```diff
@@ -10,6 +10,8 @@
 from sse import tasks
 from sse.dataclasses import SSEAccessLogs
 
+GNUPG_HOME = "/app/.gnupg"
```

Comment:
I kind of don't want to move this to settings or env var because I don't think anyone needs to change this value? What do you think?

Reply:
Agreed.